Source/Diagnostics/BTDiagnostics.H

```diff
@@ -74,6 +74,11 @@ private:
      *                is initialized.
      */
     void InitializeFieldFunctors (int lev) override;
+
+    void InitializeFieldFunctorsRZopenPMD (int lev) override;
+    /** Whether to dump the RZ modes */
+    bool m_dump_rz_modes = false;
+    void AddRZModesToOutputNames (const std::string& field, const int ncomp, bool cellcenter_data);
```

Comment:
Let's add a newline for readability and a doxygen string to `AddRZModesToOutputNames`? :)

Reply:
done